README.md

```diff
@@ -23,7 +23,7 @@ To install the bindings via [Composer](https://getcomposer.org/), add the follow
     }
   ],
   "require": {
-    "konfig/snaptrade-php-sdk": "1.12.0"
```

Comment:
do a minor bump instead of patch

Reply:
does this look right? not sure why it went from 12 to 15

Reply:
no, it should be 13, thats weird. make sure the `konfig.yaml` is 1.13.0 before generating